visualization/app/codeCharta/ui/heightSettingsPanel/heightSettingsPanel.component.spec.ts

```diff
@@ -104,6 +122,22 @@ describe("HeightSettingsPanelController", () => {
 		})
 	})
 
+	describe("onShowMetricLabelNameValueChanged", () => {
+		it("should set showMetricValue in viewModel", () => {
+			heightSettingsPanelController.onShowMetricLabelNameValueChanged(false)
+
+			expect(heightSettingsPanelController["_viewModel"].showMetricValue).toBeFalsy()
+		})
+	})
+
+	describe("onShowMetricLabelNodeNameChanged", () => {
+		it("should set showNodeName in viewModel", () => {
+			heightSettingsPanelController.onShowMetricLabelNodeNameChanged(false)
+
+			expect(heightSettingsPanelController["_viewModel"].showNodeName).toBeFalsy()
```

Comment:
I prefer toBe(false) instead of toBeFalsy(). It's just a bit more explicit and reduces the risk of false-positive tests.